### ISSUE TITLE
Attack simulator availability is confusing

### ISSUE THREAD

**Issue by asdcristi** [MicrosoftDocs/microsoft-365-docs#11503]
🚨 Looks like you arrived from the internal review site 
🚨 Do not enter Microsoft confidential information here

The below section, confuses customers, as we're including main regions, then countries within the regions:

Attack simulation is available in the following regions: NAM, APC, EUR, IND, CAN, AUS, FRA, GBR, JPN, KOR, BRA, LAM, CHE, NOR, ZAF, ARE and DEU.

Example, EUR is specified, plus NOR. This is confusing, because we don't understand if EUR is taken into consideration as a Continent, including Sweden, as EU, or EEA.

In order to shed light, a list of all countries should replace these regions, which would answer availability foreach country.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 59c87455-77af-093a-6fb6-286049aad15a
* Version Independent ID: b2690e98-781d-4a27-a9bc-bd621964feb0
* Content: [Get started using Attack simulation training - Office 365](https://review.learn.microsoft.com/en-us/microsoft-365/security/office-365-security/attack-simulation-training-get-started?view=o365-worldwide)
* Content Source: [microsoft-365/security/office-365-security/attack-simulation-training-get-started.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/office-365-security/attack-simulation-training-get-started.md)
* Service: **microsoft-365-security**
* Sub-service: **mdo**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by richa-b23** [comment]
Please find the updated details below:

Attack Simulator is available in NAM, EUR and APC regions and in following countries ARE, AUS, BRA, CAN, CHE, DEU, FRA, GBR, IND, JPN, KOR, LAM, NOR, QAT, SGP, SWE, ZAF

**Comment by chrisda** [comment]
Content updated per referenced pull requestion.

Thanks for the feedback.

Closing.

**Comment by richa-b23** [comment]
Hi @chrisda the 'also' word in the below text has caused slight confusion for a customer. 
"Attack simulation training is available in the following regions: APC, EUR, and NAM, and also in the following countries: ARE, AUS, BRA, CAN, CHE, DEU, FRA, GBR, IND, JPN, KOR, LAM, NOR, QAT, SGP, SWE, and ZAF."

Can we rephrase it to as given below:
"Attack simulation training is available in the following regions: APC, EUR, and NAM. The countries within these regions where it is available include ARE, AUS, BRA, CAN, CHE, DEU, FRA, GBR, IND, JPN, KOR, LAM, NOR, QAT, SGP, SWE, and ZAF."

**Comment by chrisda** [comment]
@richa-b23, OK. Done.
